assets/templates/layouts/layout.nunjucks

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset ="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Kyrio GO2Broadband{% block title %}{% endblock %}</title>
+        <meta name="description" content="Shop for broadband cable services in your area. See plans, prices, and schedule your service.">
```

Comment:
@KyleKrukar Do we want to use the same `<meta name="description">` for the Affiliates page as the homepage?